### PR TITLE
feat(url): remove old URLS

### PR DESCRIPTION
**Pull request by kaka-lin**
## Types of changes
**Thanks for sending a pull request! Please fill in the following content to let us know better about this change.**
Please put an `x` in the box that applies

- [ ] **Bugfix**
- [x] **New feature**
- [ ] **Refactoring**
- [ ] **Breaking change** (any change that would cause existing functionality to not work as expected)
- [ ] **Documentation Update**
- [ ] **Other (please describe)**

## Description

We're now attempted to migrate all the attendee-facing pages to our new frontend project and remain the pages related to the proposal & review system in pycon.tw backend project. Thus, remove the URLs that are deprecated.

### CODE REVIEW

**Comment by kaka-lin** [comment]
[Remove old templates in `pycon.tw`](https://trello.com/c/XwSMahbb/32-remove-old-templates-in-pycontw)

File: src/pycontw2016/urls.py [hunk 1]
```diff
@@ -32,8 +32,8 @@
 
 # These should not be prefixed with language.
 urlpatterns += [
-    url(r'^ccip/', include('ccip.urls')),
-    url(r'^api/', include('api.urls')),
+    #url(r'^ccip/', include('ccip.urls')),
+    #url(r'^api/', include('api.urls')),
```
Comment (mattwang44) [inline src/pycontw2016/urls.py:36]: please keep them

File: src/users/views.py [hunk 2]
```diff
@@ -90,7 +90,7 @@ def user_dashboard(request):
 
 @login_required
 def user_profile_update(request):
-    logout_next = reverse('index')
+    #logout_next = reverse('index')
```
Comment (mattwang44) [inline src/users/views.py:93]: should redirect to dashboard

File: src/pycontw2016/urls.py (hunk 1 above)
Comment (kaka-lin) [reply src/pycontw2016/urls.py:36]: ok

File: src/users/views.py (hunk 2 above)
Comment (kaka-lin) [reply src/users/views.py]: Done

**Comment by hsuanchi** [comment]
LGTM !

**Review by setmao — APPROVED** [review]
LGTM

**Pull request merged**